src/femr/transforms/sickkids.py

```diff
@@ -0,0 +1,31 @@
+"""Transforms that are unique to SK OMOP."""
+
+import datetime
+from typing import Optional
+
+from femr import Patient
+from femr.extractors.omop import OMOP_BIRTH
+
+
+def replace_categorical_measurement_results(patient: Patient) -> Patient:
+    """Replace value_as_number of 9999999 with None. 9999999 is the default
+    Clarity assignment when the lab result is a categorical value.
+    """
+    for event in patient.events:
+        if event.omop_table == "measurement" and event.value == 9999999:
+            event.value = None
+
+    patient.resort()
```

Comment:
Resort is technically not needed here, as you didn't fiddle with the order of events. Would make things a tiny bit faster if you did not resort here.

Reply:
Thanks! Updated.